events/EiffelActivityStartedEvent/simple.json

```diff
@@ -0,0 +1,22 @@
+{
+  "meta": {
+    "type": "EiffelActivityStartedEvent",
+    "version": "1.0",
+    "time": 1234567890,
+    "domainId": "example.domain",
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+  },
+  "data": {
+    "executionUri": "https://my.jenkins.host/myJob/43",
+    "liveLogs": [
+      {
+       "name": "logfile",
+       "uri": "file:///tmp/logs/data.log"
+      }
+    ]
+  },
+  "links": {
+    "activityExecution": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
```

Comment:
I think we should use different UUIDs in an event to indicate that they point to different events.